miden-tx/tests/integration/scripts/swap.rs

```diff
@@ -70,10 +71,10 @@ fn prove_swap_script() {
     let tx_script_target = executor
         .compile_tx_script(tx_script_code.clone(), vec![(target_pub_key, target_sk_felt)], vec![])
         .unwrap();
-    let tx_args_target = TransactionArgs::new(Some(tx_script_target), None);
+    let tx_args_target = TransactionArgs::new(Some(tx_script_target), None, AdviceMap::default());
```

Comment:
Seems like the only thing we are using transaction args for is the tx script. Why not use `TransactionArgs::with_tx_script()` constructor?